dotaservice/protos/DotaService.proto

```diff
@@ -25,8 +26,22 @@ enum HostMode {
   HOST_MODE_GUI_MENU = 2;
 }
 
+enum SelectionType {
+  SELECTION_TYPE_PICK = 0;
+  SELECTION_TYPE_BAN = 1;
+  // For Ability Draft
+  SELECTION_TYPE_ABILITY_PICK = 2;
+}
+
 message Empty {}
 
+message HeroSelection {
```

Comment:
Most important for me now is that I'd be able to set who's actually controlling the hero; the default bot or myself.